src/frontend/components/UI/SearchBar/index.css

```diff
@@ -3,8 +3,9 @@
   width: 100%;
   position: relative;
   display: inline-flex;
-  background: var(--input-background);
-  border-radius: var(--space-3xs);
+  background: var(--search-bar-background, var(--input-background));
+  border-radius: var(--space-md);
+  border: 2px solid var(--search-bar-border, var(--input-backgroundd));
```

Comment:
just personal opinion, but this border looks a bit off in some themes other than the new default, in dracula it feels `blurry`

maybe it can be added only in the new theme instead of being a default of the element?

I also think the color is too dark and the contrast is not great

Reply:
I changed it to show only when on focus, also it is not a border anymore but a box-shadow.